wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1301,6 +1302,55 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    shortcuts_category = {
+        "Actions": [
+            {"Label": "Save", "Shortcut": ("⌘" if is_mac else "Ctrl") + " + s"},
```

Comment:
Instead of repeating the logic, we can extract this to a variable, e.g.

```py
modifier = "⌘" if is_mac else "Ctrl"
```

and in this case I think the keys should be in snake case, e.g.

```suggestion
            {"label": "Save", "shortcut": f"{modifier} + s"},
```

Reply:
Done